h/schemas/api/group.py

```diff
@@ -21,6 +24,20 @@
 }
 
 
+class FilterGroupAnnotationsSchema(colander.Schema):
+    moderation_status = colander.SchemaNode(
+        colander.String(),
+        validator=colander.OneOf(["approved", "pending", "denied", "spam"]),
```

Comment:
These should match the capitalization here: https://github.com/hypothesis/h/pull/9493

Reply:
Fixed, went for upper case but I don't have a preference really